src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Extensions/ObservableRangeCollectionExtension.shared.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Xamarin.CommunityToolkit.ObjectModel
```

Comment:
@pictos then should we put it inside Xamarin.CommunityToolkit.ObjectModel.Extensions namespase?

Reply:
Visual Studio would suggest adding this namespace (I checked), so users would have no idea that this method exists.

Reply:
@AndreiMisiukevich This class and method should be invisible to our users. So in order to use it to initialize the collection and don't know about it, we need to keep this extension method in the same namespace. That way everything will works. On top of my head, we only need to document that they can do it using the full name (I believe). Something like

```csharp
var eventTypes = new Xamarin.CommunityToolkit.ObjectModel.ObservableRangeCollection<EventType>()
{
     timetableInfo.EventTypes(lessonInfo.Lesson.ID).OrderBy(et => et.Entity.ShortName)
};
```

But I'm ok with this limitation